### ISSUE TITLE
[DOC-1243] With list of base urls it is not possible to select url that will be used in test requests

### ISSUE THREAD

**Issue by marclave** [scalar/scalar#357]
### Expected Behavior

You should be able to select the URL that is default for requests

### Actual Behavior

Not possible to select url that will be used in test requests

### Reproduction steps

```bash
load a swagger/oas file with a list of base urls it is not possible to select url that will be used in test requests
```

### Version

latest

<sub>[DOC-1243](https://linear.app/api-docs/issue/DOC-1243/with-list-of-base-urls-it-is-not-possible-to-select-url-that-will-be)</sub>

**Comment by hanspagel** [comment]
Fixed in #348

**Comment by marclave** [comment]
[https://github.com/scalar/scalar/issues/357](https://github.com/scalar/scalar/issues/357)
